tests/Support/SupportCarbonTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
I assume whether or not this is valuable is based on whether or not the Carbon class above is valuable

Reply:
Yeah, those have been removed.